Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/ThermoFluid/Chiller.mo

```diff
@@ -0,0 +1,314 @@
+          lineColor={0,0,0},
+          smooth=Smooth.None,
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{-42,0},{-52,10},{-32,10},{-42,0}},
+          lineColor={0,0,0},
+          smooth=Smooth.None,
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-44,50},{-40,10}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-44,-12},{-40,-52}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{38,50},{42,-52}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{18,22},{62,-20}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{40,22},{22,-10},{58,-10},{40,22}},
+          lineColor={0,0,0},
+          smooth=Smooth.None,
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Line(points={{0,90},{0,36},{0,2},{18,2}},color={0,0,255}),
+        Line(points={{20,68},{20,74},{20,90},{90,90},{100,90}},color={0,0,255}),
+        Line(points={{62,0},{100,0}},color={0,0,255}),
+        Line(points={{0,-70},{0,-90},{100,-90}},color={0,0,255}),
+        Text(extent={{70,24},{120,10}},textString="P",textColor={0,0,127})}),
+        defaultComponentName = "Chi",
+        Documentation(info="<html>
+<p>
+This model describes a chiller with mechanical imterface and uses 
+<a href=\"modelica://Buildings.Fluid.Chillers.Carnot_y\">Buildings.Fluid.Chillers.Carnot_y</a> 
+as a base model. The governing equation of this implementation is based on the 
+relationship between the power and torque of the rotating object, which is 
+represented as follow: 
+</p>
+<p align=\"center\"><i>P&nbsp;=&nbsp;tau&nbsp;*&nbsp;W</i>
+</p>
+<p>
+Where, the <i>P</i> is power [W], <i>tau</i> is torque [N.m], and <i>W</i> is 
+angular velocity [rad/s]. 
+</p>
+<h4>Assumption and limitation</h4>
+<p>
+This implementation assumes that the compressor is centrifugal  
+and the relationship between compressor power and speed ideally follows 
+a cubic relationship.
+</p>
+<h4>Reference</h4>
+<p>
+<span style=\"font-family: Arial;\"><a name=\":2k\" href=\"https://ieeexplore.ieee.org/abstract/document/8598849\">
+Oliveira, Felipe, and Abhisek Ukil. &quot;Comparative performance analysis of 
+induction and synchronous reluctance motors in chiller systems for energy 
+efficient buildings.&quot;&nbsp;<i>IEEE Transactions on Industrial Informatics</i>&nbsp;15.8 (2019): 4384-4393.</a></span>
+</p>
+<p>
+<span style=\"font-family: Arial;\"><a href=\"https://www.proquest.com/docview/2414053191?pq-origsite=gscholar&fromopenview=true\">
```

Comment:
You may delete `&fromopenview=true`. This will cause html check error:
```
line 67 column 110 - Warning: unescaped & or unknown entity "&fromopenview"
```
The hyperlink will still be valid.